src/Cake.Core.Tests/Unit/Scripting/Processors/LoadDirectiveProcessorTests.cs

```diff
@@ -69,6 +69,25 @@ public void Should_Process_Multiple_Script_References_Found_In_Source(string sou
             Assert.Equal("/Working/other.cake", result.Script.Includes[1].Path.FullPath);
         }
 
+        [Theory]
+        [InlineData("#l \"%CAKE_TEST_SCRIPT_PATH%/utils.cake\"")]
+        public void Should_Process_Environment_Variable_Found_In_Source(string source)
```

Comment:
This tests fails on MacOS / Linux

```
[xUnit.net 00:00:02.7407740]     Cake.Core.Tests.Unit.Scripting.Processors.LoadDirectiveProcessorTests.Should_Process_Environment_Variable_Found_In_Source(source: "#l \"%CAKE_TEST_SCRIPT_PATH%/utils.cake\"") [FAIL]
[xUnit.net 00:00:02.7444020]       Assert.Equal() Failure
[xUnit.net 00:00:02.7445470]       Expected: 1
[xUnit.net 00:00:02.7446970]       Actual:   0
[xUnit.net 00:00:02.7463720]       Stack Trace:
[xUnit.net 00:00:02.7488290]         /Users/vagrant/git/src/Cake.Core.Tests/Unit/Scripting/Processors/LoadDirectiveProcessorTests.cs(84,0): at Cake.Core.Tests.Unit.Scripting.Processors.LoadDirectiveProcessorTests.Should_Process_Environment_Variable_Found_In_Source(String source)
Failed   Cake.Core.Tests.Unit.Scripting.Processors.LoadDirectiveProcessorTests.Should_Process_Environment_Variable_Found_In_Source(source: "#l \"%CAKE_TEST_SCRIPT_PATH%/utils.cake\"")
Error Message:
 Assert.Equal() Failure
Expected: 1
Actual:   0
Stack Trace:
   at Cake.Core.Tests.Unit.Scripting.Processors.LoadDirectiveProcessorTests.Should_Process_Environment_Variable_Found_In_Source(String source) in /Users/vagrant/git/src/Cake.Core.Tests/Unit/Scripting/Processors/LoadDirectiveProcessorTests.cs:line 84
```

Reply:
I fixed the test. Now there is a CodeFactor issue which is not really connected to my PR, i think.